pkg/collector/adapters/config_to_ports_test.go

```diff
@@ -144,6 +144,39 @@ service:
 	}
 }
 
+func TestReceiverPortsSortedCorrectly(t *testing.T) {
```

Comment:
Thanks for the test! 

It would be better to change the `TestExtractPortsFromConfig` because it does the same thing - check what ports are returned and just change how the assert is written in that function (see my other comment).